generators/python/urllib3/templates/setup.mustache

```diff
@@ -17,7 +17,7 @@ PYTHON_REQUIRES = ">=3.7"
 {{#apis}}
 {{#-last}}
 REQUIRES = [
-    "urllib3 >= 1.25.3, < 2.1.0",
+    "urllib3 >= 1.25.3",
```

Comment:
LGTM, let just add a comment to not forget about this change in case template got refreshed:
```suggestion
    "urllib3 >= 1.25.3",   {{! upper bound not needed, < 2.1.0 removed }}
```